functorch/csrc/BatchRulesReduceOps.cpp

```diff
@@ -285,6 +380,10 @@ TORCH_LIBRARY_IMPL(aten, FT_BATCHED_KEY, m) {
   REDUCTION_BOXED(any.dim);
   REDUCTION_BOXED(argmax);
   REDUCTION_BOXED(argmin);
+  VMAP_SUPPORT("cdist", cdist_batch_rule);
```

Comment:
cdist is a composite operation, right? Why didn't we just do an `OP_DECOMPOSE` for it?

Reply:
Let me check that, I haven't thought about that. Thanks for the suggestion.